Experiences/Experiences/Helpers/ShiftableViewController.swift

```diff
@@ -0,0 +1,124 @@
+//
+//  ShiftableViewController.swift
+//  Experiences
+//
+//  Created by Craig Swanson on 3/22/20.
+//  Copyright © 2020 craigswanson. All rights reserved.
+//
+
+import UIKit
+
```

Comment:
Would have liked to have seen more use of extensions here. Not the worst I've seen but something to keep in mind.